lib/common.sh

```diff
@@ -176,8 +176,8 @@ export MARIADB_IMAGE_REPO="${MARIADB_IMAGE_REPO:-https://github.com/metal3-io/ma
 export MARIADB_IMAGE_BRANCH="${MARIADB_IMAGE_BRANCH:-main}"
 export MARIADB_IMAGE_COMMIT="${MARIADB_IMAGE_COMMIT:-HEAD}"
 
-export BUILD_CAPM3_LOCALLY="${BUILD_CAPM3_LOCALLY:-false}"
-export BUILD_IPAM_LOCALLY="${BUILD_IPAM_LOCALLY:-false}"
+export BUILD_CAPM3_LOCALLY="${BUILD_CAPM3_LOCALLY:-true}"
+export BUILD_IPAM_LOCALLY="${BUILD_IPAM_LOCALLY:-true}"
```

Comment:
Please check these variables. It should be true for local run only.

Reply:
it was a leftover. Thank you for noticing. Fixed

Reply:
It is safer to export this kind of test configuration changes in `config_example.sh`, then they don't get accidentally merged.